src/modules/bootloader/main.py

```diff
@@ -502,6 +502,35 @@ def get_kernels(installation_root_path):
     return kernel_list
 
 
+def install_clr_boot_manager():
+    """
+    Installs clr-boot-manager as the bootloader for EFI systems
+    """
+    libcalamares.utils.debug("Bootloader: clr-boot-manager")
+
+    installation_root_path = libcalamares.globalstorage.value("rootMountPoint")
+    kernel_dir = os.path.join(installation_root_path, "etc", "kernel", "cmdline.d")
+    kernel_resume_file = os.path.join(kernel_dir, "10_resume.conf")
+
+    partitions = libcalamares.globalstorage.value("partitions")
+    swap_uuid = ""
+
+    # Get the UUID of the swap partition, if present
+    for partition in partitions:
+        if partition["fs"] == "linuxswap" and not partition.get("claimed", None):
+            continue
+        has_luks = "luksMapperName" in partition
+        if partition["fs"] == "linuxswap" and not has_luks:
+            swap_uuid = partition["uuid"]
```

Comment:
Shouldn't we call `get_kernel_params()` here instead?  That would build a set of params that would work for many different scenarios and also honor the conf file options.